assets/src/stories-editor/components/block-navigation/edit.css

```diff
@@ -4,19 +4,41 @@
 	left: 45px;
 	top: 75px;
 	width: 300px;
+	min-width: 115px;
 	z-index: 80;
 	list-style-type: none;
 	padding: 0;
 	margin: 0;
 }
 
+@media (max-width: 1200px) {
+
+	.is-sidebar-opened #amp-story-block-navigation {
+		left: 0;
+	}
+}
+
+@media (max-width: 1100px) and (min-width: 1001px) {
```

Comment:
This isn't wrong, but do you think having the `min-width` before the `max-width` would make it easier to read?

Reply:
Sure, I put it this way thinking of desktop-first, but it's not following this generally. Will change.